drawers/cOrderDrawer.cpp

```diff
@@ -17,20 +18,19 @@ cOrderDrawer::~cOrderDrawer() {
 }
 
 void cOrderDrawer::drawOrderPlaced() {
-	BITMAP *bmp_trans=create_bitmap(buttonBitmap->w, buttonBitmap->h);
-	clear_to_color(bmp_trans, makecol(255,0,255));
+  // Create a temporary bitmap that destroys itself when going out of scope
+  std::unique_ptr<BITMAP, decltype(&destroy_bitmap)> bmp_trans{create_bitmap(buttonBitmap->w, buttonBitmap->h), destroy_bitmap};
```

Comment:
lets move this line into the constructor so we can re-use the bitmap, and only destroy it once we destroy `cOrderDrawer` ?